packages/search-input/src/SearchResultsMenu/SearchResultsMenu.tsx

```diff
@@ -26,7 +26,7 @@ const MAX_MENU_HEIGHT = 256;
 export const SearchResultsMenu = React.forwardRef<
   HTMLUListElement,
   SearchResultsMenuProps
->(({ children, open = false, refEl }: SearchResultsMenuProps, ref) => {
+>(({ children, open = false, refEl, ...rest }: SearchResultsMenuProps, ref) => {
```

Comment:
how do we know that rest is all popover props? what is rest / should we just pull out the props that we are trying to pass to popover explicitly since those are known

Reply:
The type was extending `PortalControlProps` which is also extended by `PopoverProps`. I did notice this also extends `HTMLElementProps<'ul'>`, so I took it out and made it explicit